### DISCUSSION TITLE
Displaying separate UMAPs for samples processed together

### DISCUSSION THREAD

**Discussion by sylestiel** [GreenleafLab/ArchR#952] [Questions / Documentation] [answered]
Is there a way to generate **separate** UMAPs for each of the component samples (when looking for a given marker) when multiple scATAC samples are processed together?

Thanks!

**Comment by rcorces** [comment, marked as answer]
I'm not sure I fully understand. Perhaps you are looking for the `highlightCells` parameter?

**Comment by sylestiel** [comment]
This has to do with plotEmbedding function. If I have multiple arrow files combined into a single project but want to see them in separate UMAPs side by side (basically like splitting them to view them simultaneously) is that possible?

**Reply by rcorces** [reply]
Did you try my suggestion using `highlightCells`?

`plotEmbedding(ArchRProj = projHeme5, embedding = "UMAP", colorBy = "cellColData", name = "Sample")`
![image](https://user-images.githubusercontent.com/10343046/127901510-38953dc7-4eb3-4147-866d-188653158371.png)

`plotEmbedding(ArchRProj = projHeme5, embedding = "UMAP", colorBy = "cellColData", name = "Sample", highlightCells = getCellNames(ArchRProj = projHeme5)[which(as.character(projHeme5@cellColData$Sample) == "scATAC_BMMC_R1")])`
![image](https://user-images.githubusercontent.com/10343046/127901584-21bfa2d6-e014-4184-88ea-a587690ce06f.png)

**Comment by sylestiel** [comment]
I did not know how to use the highlight parameter initially.  Thanks for providing this example.
